packages/gatsby-recipes/src/renderer/render.js

```diff
@@ -253,48 +257,53 @@ const render = (recipe, cb, inputs = {}, isApply, isStream, name) => {
   const renderResources = isDrained => {
     result = RecipesReconciler.render(recipeWithWrapper, plan, name)
 
-    const resources = transformToPlan(result)
+    const resourcesArray = transformToPlan(result)
 
     const isDone = () => {
+      let result
```

Comment:
Was there a reason for refactoring the return early out? Adds lots of lines to the diff but it seems to me like it doesn't change anything.

Reply:
just it was easier for me to double-check the logic this way